dashboard/app/helpers/ai_system_prompts/system_prompt_helper.rb

```diff
@@ -0,0 +1,101 @@
+require 'cdo/shared_constants'
+
+module AiSystemPrompts::SystemPromptHelper
+  # What's shared between Evaluate & AI Tutor?
+  def self.get_basic_system_prompt(level, unit)
+    base_prompt =
+      "You are an expert Computer Science teacher. Your students are in grades: #{get_grade_levels(unit)}. The programming language they are learning is #{get_programming_language(unit)}. They are working on a level where they have been asked to #{get_level_instructions(level)}."
+    if programming_level?(level)
+      base_prompt += get_starter_code(level)
+      base_prompt += get_validated_level_test_file_contents(level)
+    end
+    base_prompt
+  end
+
+  def self.get_grade_levels(unit)
+    unit.get_course_version.course_offering.grade_levels
+  end
+
+  # TODO: see if there's a better way to do this maybe via marketing info for course catalog?
+  # TODO: also see if there's a unit description that would be helpful. There's one for courses,
+  # but it might be too broad.
+  def self.get_programming_language(unit)
+    if unit.csa?
+      "Java"
+    elsif unit.csp?
+      "JavaScript"
+    else
+      "Python"
+    end
+  end
+
+  def self.programming_level?(level)
+    level_type = level.type
+    # This is a subset of programming level types that also have an AI feature that uses the system prompt helper.
+    ["Applab", "Javalab", "Pythonlab"].include?(level_type)
```

Comment:
(this can be a follow up): I wonder about a future scenario where we add another level type to ai tutor and have to know to update this list. Do we have a constant already that stores which labs use ai tutor? Could we store a similar constant for programming levels that use ai tutor in the same place? Or can this method not care if a level has ai support already (presumably we would never get here if it didn't), and be robust for all our level types?